third-party/realdds/include/realdds/topics/control/control-msg.h

```diff
@@ -22,31 +22,31 @@ class control
 #pragma pack( push, 1 )
     enum class control_type : uint16_t
```

Comment:
Rename to `msg_type` same as `notification::msg_type`

Reply:
done